src/foam/nanos/theme/services.jrl

```diff
@@ -43,7 +43,7 @@ p({
 p({
   "class": "foam.nanos.boot.NSpec",
   "name": "themes",
-  "serve": true,
+  "serve": false,
```

Comment:
This suggest, this entire nspec is not needed.

Reply:
It is used in several places: 
https://github.com/kgrgreer/foam3/search?q=.get%28%22themes%22%29